core/src/main/scala/io/finch/Error.scala

```diff
@@ -1,28 +1,44 @@
 package io.finch
 
+import scala.compat.Platform.EOL
 import scala.reflect.ClassTag
 
+import cats.data.NonEmptyList
+import io.finch.items.RequestItem
+
 /**
  * A basic error from a Finch application.
  */
-trait Error extends Exception
+trait Error extends Exception {
+  def canEqual(other: Any): Boolean = other.isInstanceOf[Error]
 
-object Error {
+  override def hashCode(): Int = (super.hashCode, getMessage).##
+
+  override def equals(other: Any): Boolean = other match {
+    case that: Error => that.canEqual(this) && that.getMessage == this.getMessage
+    case _ => false
+  }
+}
 
-  import items._
+object Error {
 
   def apply(message: String): Error = new Error {
     override def getMessage: String = message
   }
 
+  object RequestErrors {
+    @deprecated("Use Multiple instead", "0.11")
+    def unapply(e: Throwable): Option[Seq[Throwable]] = Some(Seq(e))
```

Comment:
Ah wait, I was confused for a 'generic' Error unapply. But this should only handle Multiple then. Don't mind my previous comment..